raiden/api/rest.py

```diff
@@ -75,19 +75,26 @@
 
 URLS_V1 = [
     ('/address', AddressResource),
-    ('/channels', ChannelsResource),
-    ('/channels/<hexaddress:channel_address>', ChannelsResourceByChannelAddress),
-    ('/tokens', TokensResource),
-    ('/tokens/<hexaddress:token_address>/partners', PartnersResourceByTokenAddress),
-    ('/tokens/<hexaddress:token_address>', RegisterTokenResource),
-    ('/events/network', NetworkEventsResource),
+    ('/channels/<hexaddress:registry_address>', ChannelsResource),
+    (
+        '/channels/<hexaddress:registry_address>/<hexaddress:channel_address>',
+        ChannelsResourceByChannelAddress
+    ),
+    ('/tokens/<hexaddress:registry_address>', TokensResource),
+    (
+        '/tokens/<hexaddress:registry_address>/<hexaddress:token_address>/partners',
+        PartnersResourceByTokenAddress
+    ),
+    ('/tokens/<hexaddress:registry_address>/<hexaddress:token_address>', RegisterTokenResource),
+    ('/events/<hexaddress:registry_address>/network', NetworkEventsResource),
     ('/events/tokens/<hexaddress:token_address>', TokenEventsResource),
     ('/events/channels/<hexaddress:channel_address>', ChannelEventsResource),
     (
-        '/transfers/<hexaddress:token_address>/<hexaddress:target_address>',
+        '''/transfers/<hexaddress:registry_address>/
+        <hexaddress:token_address>/<hexaddress:target_address>''',
```

Comment:
triple quotes don't automatically remove the space, so this is URL would be:


    /transfers/registry/                                    token/target